From a54206722d075b773801f8663bd50b14cb4b56c6 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 18 Jan 2018 08:55:34 +0100
Subject: [PATCH] dracut.cmdline.7.asc: document rd.ccw and rd.znet

rd.znet should be used instead of rd.ccw.

Add more info about rd.znet.

Fixes github issue 364
---
 dracut.cmdline.7.asc | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/dracut.cmdline.7.asc b/dracut.cmdline.7.asc
index 8f83405e..8f86efe0 100644
--- a/dracut.cmdline.7.asc
+++ b/dracut.cmdline.7.asc
@@ -811,6 +811,9 @@ rd.zfcp=0.0.4000
 ZNET
 ~~~~
 **rd.znet=**__<nettype>__,__<subchannels>__,__<options>__::
+    The whole parameter is appended to /etc/ccw.conf, which is used on
+    RHEL/Fedora with ccw_init, which is called from udev for certain
+    devices on z-series.
     rd.znet can be specified multiple times on the kernel command line.
 +
 [listing]
@@ -1083,7 +1086,8 @@ their new replacement.
 
 rdbreak:: rd.break
 
-rd_CCW:: rd.ccw
+rd.ccw:: rd.znet
+rd_CCW:: rd.znet
 
 rd_DASD_MOD:: rd.dasd
 
